Library/Homebrew/cmd/prune.rb

```diff
@@ -1,5 +1,6 @@
 require "keg"
 require "cmd/tap"
+require "cmd/unlinkapps"
```

Comment:
I don't feel super-strongly about this but we're trying to avoid `cmd/` includes now so if you're feeling extra lovely (can be a future PR) pulling these methods into another module would be :heart: